### ISSUE TITLE
Unable to install any free ADO extension from marketplace. Server status 500 error

### ISSUE THREAD

**Issue by johleung3838** [microsoft/vsmarketplace#302]
Steps to reproduce
1. Go to marketplace
2. Search for Katalon or any other ADO extension
3. Click on the button to get the extension
4. Status 500 code is displayed
![500 status error](https://user-images.githubusercontent.com/96275738/149428314-9711a6a5-010f-456d-922a-1d5c8999a787.PNG)

More information about this error
GetUserAccessToken: Failed to obtain an access token of identity 99b76673-269f-7cbb-af1a-463c352c1f43. AAD returned silent failure.

Additional technical details:
Activity ID: a1122d40-0dda-4613-b49c-d21ea29387a6
Date and Time: 13/01/2022 23:58:49 (UTC)

**Comment by madhurivadaligithub** [comment]
Thanks for contacting VS Marketplace Support Team. Could you please try the following steps to unblock.
 
Approach 1.
	a. Logout from all the active session of Azure DevOps organization using https://aka.ms/vssignout 
	b. Open a new in-private window and Navigate to https://aka.ms/vssignout?wa=wsignoutcleanup1.0 and let it complete (You should see a tiny checkmark and nothing else). 
	c. Now visit marketplace.visualstudio.com and start installing.
 
Approach 2. 
Try to download the extension in incognito mode or clear all Marketplace cookies on your browser. 
To clear the session cookies (for chromium based browsers): 
	a. Click on the lock icon in the address bar, select Cookies 
	b. Select marketplace.visualstudio.com in the dialog and click remove and refresh the page 

Regards,
Madhuri

**Comment by madhurivadaligithub** [comment]
@johleung3838  Is this issue resolved?

**Comment by johleung** [comment]
> @johleung3838 Is this issue resolved?

Yes, issue is related specifically to our company blocking installation of ADO extensions in our ADO organisation.

**Comment by SaiKanth007** [comment]
Closing this ticket since the issue is resolved.
